ennead/utils/markdown.py

```diff
@@ -31,13 +31,24 @@ def handleMatch(self, m, data):
         return elem, m_start, index
 
 
-class DictCache(dict):
+class DictCache:
     """Dict with .set() method auto-encoding values"""
+    cache: Dict[str, bytes]
 
-    def set(self, key: str, value: str, **_: Any) -> None:
+    def __init__(self):
+        self.cache = {}
```

Comment:
(можешь не обращать внимания, это моя личная вкусовщина)
```suggestion
        self.cache = dict()
```